### PR TITLE
Add support for the beaglebone green board

### PR DESCRIPTION
**Pull request by floion**
@agherzan @michal-mazurek @telphan @lorenzo-stoakes 

Connect to #28

### CODE REVIEW

**Comment by floion** [comment]
this needs some love

**Comment by agherzan** [comment]
Loving it now.

File: beaglebone-green.coffee [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
+
+module.exports =
+	slug: 'beaglebone-green'
+	name: 'BeagleBone Green'
+	arch: 'armv7hf'
+	state: 'experimental'
```
Comment (agherzan) [inline beaglebone-green.coffee]: Is this still experimental?

**Comment by agherzan** [comment]
LGTM

**Comment by floion** [comment]
fixes #28

File: beaglebone-green.coffee (hunk 1 above)
Comment (floion) [inline beaglebone-green.coffee]: I think it should pass at least one release testing cycle before changing the experimental flag. But no strong opinion on that and can remove it if we want to do that

Comment (agherzan) [inline beaglebone-green.coffee]: Makes sense. Ack.

**Pull request merged**